docs/README.md

```diff
@@ -60,58 +60,59 @@ The Media Communications Mesh (MCM) enables efficient, low-latency media transpo
 1. **Clone the repository**
 
    ```bash
-   $ git clone https://github.com/OpenVisualCloud/Media-Communications-Mesh.git
+   git clone https://github.com/OpenVisualCloud/Media-Communications-Mesh.git
    ```
 
 2. **Navigate to the Media-Communications-Mesh directory**
 
     ```bash
-    $ cd Media-Communications-Mesh
+    cd Media-Communications-Mesh
     ```
 
-3. **Install Dependencies**, choose between options `a)` or `b)`.
+3. **Install Dependencies**, choose between **OPTION A** or **OPTION B**.
 
-    a) Use all-in-one environment preparation script. The script was tested under `Ubuntu 20.04`, `Ubuntu 22.04`, `Ubuntu 24.04`, `CentOS Stream8`, `CentOS Stream9` and kernel version `5.15` environments.
+    - **OPTION A:** Use all-in-one environment preparation script. The script was tested under `Ubuntu 20.04`, `Ubuntu 22.04`, `Ubuntu 24.04`, `CentOS Stream8`, `CentOS Stream9` and kernel version `5.15` environments.
+        To use this option run the following command:
 
-    To use this option run the following command:
+        ```bash
+        sudo ./scripts/setup_build_env.sh
```

Comment:
I've noticed that new script was added `setup_ice_irdma.sh`.

I don't follow all the changes in scrips but it seems that somebody remove installation of drivers from `setup_build_env.sh` to this new script, so now it is required to invoke both if we want to install all. ( to be confirmed)

Could you look into it? Add add information about this new script if it is required?

Reply:
Sure, I will take care of that.

Ref: #267
Asked @Mionsz for a comment on PM.

Reply:
Fixed that by adding the script to the mix.

Reply:
sorry to bother you, but the drivers has to be installed before `setup_build_env.sh`

Reply:
You're absolutely right! Thanks! :)